pkg/services/authn/clients/ext_jwt_test.go

```diff
@@ -118,7 +118,7 @@ var (
 	}
 	invalidSubjectIDTokenClaims = idTokenClaims{
 		Claims: jwt.Claims{
-			Subject:  "service-account:2",
+			Subject:  "anonymous:2",
```

Comment:
Perhaps we should add a valid option with the `service-account` subject to exercise the fact that this has become valid supported?